docs/modules/user-guide/partials/con_using-a-top-level-attribute.adoc

```diff
@@ -0,0 +1,38 @@
+[id="con_using-top-level-attribute_{context}"]
+= Using a top-level attribute
+
+[role="_abstract"]
+
+Use top-level attributes in devfiles to reuse global attributes in different sections of the same devfile.
+
+.Devfile variable
+
+The devfile 2.x specification includes global variables which define key value pairs.
```

Comment:
1. See the devfile 2.x specification that includes global variables which define key value pairs:
```yaml
variables:
  my-var: "some value"
```
2. Use `{{my-var}}` to reference the variables in non-string fields within the devfile.
3. Set only the variable field to make different template versions of your devfile.

Reply:
Note, when you have all these action verbs in your concept module, it's a good sign you're actually writing a procedure module because you're given the users lots of steps to complete :)